### PR TITLE
Tweaks on chat

### PR DESCRIPTION
**Pull request by cecileboucheron**
@keybase/react-hackers

### CODE REVIEW

File: shared/chat/conversation/messages/message-popup/exploding/index.js [hunk 1]
```diff
@@ -82,38 +82,43 @@ class ExplodingPopupHeader extends React.Component<PropsWithTimer<Props>, State>
...
-              onUsernameClicked="profile"
-              colorFollowing={true}
-              colorYou={true}
-              usernames={[author]}
-              underline={true}
-              type="BodySmallSemibold"
-            />
+        <Box2
+          direction="vertical"
+          style={{paddingLeft: globalMargins.small, paddingRight: globalMargins.small}}
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/message-popup/exploding/index.js]: all styles like this should be in the stylessheetcreate below

File: shared/chat/conversation/messages/message-popup/exploding/index.js [hunk 2]
```diff
@@ -82,38 +82,43 @@ class ExplodingPopupHeader extends React.Component<PropsWithTimer<Props>, State>
...
+        >
+          <Box2 direction="vertical">
+            <Text type="BodySmall" style={{color: globalColors.black_75}}>
+              EXPLODING MESSAGE
+            </Text>
+          </Box2>
+          <Box2 direction="horizontal">
+            <Text type="BodySmall">by</Text>
+            <Box2 direction="horizontal" gap="xtiny" gapStart={true} style={{alignItems: 'center'}}>
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/message-popup/exploding/index.js]: same here

File: shared/chat/conversation/messages/message-popup/payment/index.js [hunk 3]
```diff
@@ -68,7 +66,12 @@ const Header = (props: HeaderProps) =>
           </Kb.Text>
         )}
       </Kb.Box2>
-      <Kb.Box2 direction="vertical" fullWidth={true} centerChildren={true}>
+      <Kb.Box2
+        direction="vertical"
+        fullWidth={true}
+        centerChildren={true}
+        style={{paddingLeft: Styles.globalMargins.small, paddingRight: Styles.globalMargins.small}}
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/message-popup/payment/index.js]: same here

**Comment by cecileboucheron** [comment]
@chrisnojima updated

File: shared/chat/conversation/messages/message-popup/exploding/index.js [hunk 4]
```diff
@@ -186,6 +183,11 @@ const ExplodingPopupMenu = (props: PropsWithTimer<Props>) => {
 
 const oneMinuteInS = 60
 
+const styleMessageInfoContainer = {
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/message-popup/exploding/index.js]: we should use styleSheetCreate and not plain consts anymore

**Comment by cecileboucheron** [comment]
@chrisnojima bump ^

**Pull request merged**